.gitignore

```diff
@@ -4,3 +4,4 @@ public
 .DS_Store
 lib
 dist
+storybook-static
```

Comment:
Since we're adding this to the `.gitignore`, can you also remove the last section about `storybook-static` from the README?

Reply:
Is the issue that https://github.com/NYPL/nypl-design-system/pull/71 resolves no longer relevant?

Reply:
Looks so because it'll be running those two other scripts is my guess

Reply:
Yea I just read through that PR. It _should_ now be possible to gitignore this folder, but I will have to double check once we merge this that it worked (I think it will)